index.html

```diff
@@ -9,31 +9,32 @@
     <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet"/>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="style.css" />
+    <script src="app.js" defer></script>
   </head>
   <body>
     <header class="head">
       <div class="header container">
         <div class="logo">
-          <h3>Dave Mba</h3>
+          <h3>Dave Mba.</h3>
         </div>
         <div class="nav">
           <nav>
+            <i class="fa-solid fa-xmark xmark"></i>
             <ul>
-              <li><a href="">Portfolio</a></li>
-              <li><a href="">About</a></li>
-              <li><a href=""> Contact</a></li>
+              <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+              <li><hr></li>
+              <li><a href="#about-me" class="nav-link">About</a></li>
+              <li><hr></li>
+              <li><a href="#contact-us" class="nav-link"> Contact</a></li>
+              <li><hr></li>
               <li><i class="ri-mail-line"></i></li>
             </ul>
           </nav>
         </div>
```

Comment:
- [ ] Please know that the mobile menu should cover all of the screen, kindly fix this:

![image](https://user-images.githubusercontent.com/20567503/180893076-537c9830-9473-4d0e-84f0-939c46cc7a31.png)